cypress/e2e/iframe.cy.ts

```diff
@@ -0,0 +1,17 @@
+import {IFramePage} from "../page";
+
+describe("Check the iframe", () => {
+  let iFramePage: IFramePage;
+
+  before(() => {
+    iFramePage = new IFramePage;
+  });
+
+  it("Goes to page and checks iFrame exists", () => {
+    iFramePage.visit();
+
+    iFramePage.getFrameTitle("HTML Tutorial");
+
+    iFramePage.goToCssPageInFrame("CSS Tutorial");
+  });
```

Comment:
this class should have two tests, instead of one. The first test should verify the html tutorial title, the second one the css tutorial title, as two tests are required, the visit can go on the before hook